app/src/main/java/social/entourage/android/new_v8/events/details/feed/FeedFragment.kt

```diff
@@ -174,6 +183,60 @@ class FeedFragment : Fragment() {
         }
     }
 
+    private fun handleSettingsButton() {
+        binding.iconSettings.setOnClickListener {
+            with(event) {
+                eventUI = SettingUiModel(
+                    id, title,
+                    membersCount,
+                    metadata?.displayAddress,
+                    interests,
+                    description,
+                    members,
+                    member,
+                    EntourageApplication.me(context)?.id == author?.userID,
+                    recurrence
+                )
+                SettingsModalFragment.newInstance(eventUI)
+                    .show(parentFragmentManager, SettingsModalFragment.TAG)
+            }
+        }
+    }
+
+
+    private fun handleAboutButton() {
+        binding.more.setOnClickListener {
+            AnalyticsEvents.logEvent(
+                AnalyticsEvents.ACTION_GROUP_FEED_MORE_DESCRIPTION
+            )
+            val eventUI = EventUiModel(
+                eventId,
+                event.title,
+                event.membersCount,
+                event.displayAddress,
+                event.interests,
+                event.description,
+                event.members,
+                event.member,
+                EntourageApplication.me(activity)?.id == event.author?.userID,
+                event.online,
+                event.metadata,
+                event.eventUrl,
+                event.createdAt,
+                event.updatedAt,
+                event.recurrence,
+                event.neighborhoods,
+                event.location
+            )
```

Comment:
un gros with :)

Reply:
Normalement ça devrait pas être fait dans le fragment ce mapping event to eventUI